src/modules/authentication/api/userinfo.response.ts

```diff
@@ -75,5 +79,6 @@ export class UserinfoResponse {
         this.locale = info.locale;
         this.phone_number = info.phone_number;
         this.updated_at = info.updated_at;
+        this.personId = person?.id;
```

Comment:
Die dem Konstruktor übergeben person kann null oder nicht vorhanden (undefined) sein, in beiden Fällen wird das this.person assignte Ergebnis aber immer undefined sein. Eine übergebene null wird damit nicht gesetzt sondern in undefined gemapt. Ist das so gewollt?

Reply:
Ja